### PR TITLE
optimize: add application data configuration items

### PR DESCRIPTION
**Pull request by Bughue**

### CODE REVIEW

**Comment by Bughue** [comment]
> optimize : rm appdata size limit seata/seata#4473

Is 2.x branch supposed to correspond to 2.0.0 ?

File: i18n/en/docusaurus-plugin-content-docs/current/user/configurations.md [hunk 1]
```diff
@@ -177,6 +177,8 @@ transport.enable-client-batch-send-request、client.log.exceptionRate
 | metrics.registryType | Indicator registrar type | The indicator registrar type used by Metrics is a built-in compact (simple) implementation by default. Meters in this implementation only use a limited memory count, and the performance is high enough to meet most scenarios; Currently, only one indicator registrar can be set|
 | metrics.exporterList | Index result Measurement data outputter list | default prometheus. Multiple outputters are separated by English commas, such as "prometheus, jmx". Currently, only the prometheus outputters are connected|
 | metrics.exporterPrometheusPort | prometheus exporter client port number | 9898 by default|
+| server.applicationDataLimitCheck     |  enable checking of received client data | Default false |
```
Comment (funky-eyes) [inline i18n/en/docusaurus-plugin-content-docs/current/user/configurations.md:180]: Put them by type, and put them together if they start with server.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/user/configurations.md [hunk 2]
```diff
@@ -151,6 +151,8 @@ transport.enable-client-batch-send-request、client.log.exceptionRate
 | metrics.registryType                     | 指标注册器类型    |Metrics使用的指标注册器类型，默认为内置的compact（简易）实现，这个实现中的Meter仅使用有限内存计数，性能高足够满足大多数场景；目前只能设置一个指标注册器实现 |
 | metrics.exporterList                     | 指标结果Measurement数据输出器列表   |默认prometheus，多个输出器使用英文逗号分割，例如"prometheus,jmx"，目前仅实现了对接prometheus的输出器 |
 | metrics.exporterPrometheusPort          | prometheus输出器Client端口号   |默认9898 |
+| server.applicationDataLimitCheck     |是否开启应用数据大小检查 | 默认false |
```
Comment (funky-eyes) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/user/configurations.md:154]: ibid

**Review by funky-eyes — APPROVED** [review]
LGTM

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**